src/pages/workspace/accounting/qbo/import/QuickbooksChartOfAccountsPage.tsx

```diff
@@ -1,74 +1,63 @@
         >
-            <View style={[styles.flexRow, styles.mb4, styles.alignItemsCenter, styles.justifyContentBetween]}>
-                <View style={styles.flex1}>
-                    <Text fontSize={variables.fontSizeNormal}>{translate('workspace.accounting.import')}</Text>
-                </View>
-                <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
-                    <Switch
-                        accessibilityLabel={translate('workspace.accounting.accounts')}
-                        isOn
-                        disabled
-                        onToggle={() => {}}
-                    />
-                </View>
-            </View>
+            <ToggleSettingOptionRow
+                title={translate('workspace.accounting.import')}
+                switchAccessibilityLabel={translate('workspace.accounting.accounts')}
+                shouldPlaceSubtitleBelowSwitch
+                isActive
+                onToggle={() => {}}
+                showLockIcon
+            />
             <MenuItemWithTopDescription
                 interactive={false}
                 title={translate('workspace.common.categories')}
                 description={translate('workspace.common.displayedAs')}
-                wrapperStyle={styles.sectionMenuItemTopDescription}
+                wrapperStyle={[styles.sectionMenuItemTopDescription, styles.mt2]}
             />
             <Text style={styles.pv5}>{translate('workspace.qbo.accountsSwitchTitle')}</Text>
-            <View style={[styles.flexRow, styles.mb2, styles.alignItemsCenter, styles.justifyContentBetween]}>
-                <View style={styles.flex1}>
-                    <Text fontSize={variables.fontSizeNormal}>{translate('workspace.common.enabled')}</Text>
-                </View>
-                <OfflineWithFeedback pendingAction={pendingFields?.enableNewCategories}>
-                    <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
-                        <Switch
-                            accessibilityLabel={translate('workspace.accounting.accounts')}
-                            isOn={!!enableNewCategories}
-                            onToggle={() =>
-                                Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.ENABLE_NEW_CATEGORIES, !enableNewCategories)
-                            }
-                        />
-                    </View>
-                </OfflineWithFeedback>
-            </View>
-            <View style={styles.flex1}>
-                <Text style={styles.mutedTextLabel}>{translate('workspace.qbo.accountsSwitchDescription')}</Text>
-            </View>
+            <ToggleSettingOptionRow
+                title={translate('workspace.common.enabled')}
+                subtitle={translate('workspace.qbo.accountsSwitchDescription')}
+                switchAccessibilityLabel={translate('workspace.qbo.accountsSwitchDescription')}
```

Comment:
Sorry! I added the comment under the wrong line 🤦 `switchAccessibilityLabel` is different than before

Reply:
haha okay, however good catch!